package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "material-angular-dashboard",
+  "version": "0.0.0",
+  "description": "Free dashboard template with Material Design Lite and Angular",
+  "keywords": [
+    "dashboard",
+    "admin template",
+    "mdl",
+    "getmdl",
+    "Material Design",
+    "Material Design Lite",
+    "Dark",
+    "dark material design",
+    "angular",
+    "ng"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "lint": "ng lint"
+  },
+  "private": true,
```

Comment:
add [pre-commit](https://www.npmjs.com/package/pre-commit)
example using: 
`"pre-commit": [
    "lint"
  ]`
where `lint` it's a rule from script, that run lint and stylelint

Reply:
✔️